t/opcode-coverage.neon

```diff
@@ -0,0 +1,594 @@
+FUNCTION pushi()
+    VAR z: POINTER TO int
+    z := NIL
+END FUNCTION
+
+#    LOADB,      // load boolean
+FUNCTION loadb(): Boolean
+    VAR r: Boolean := TRUE
+    RETURN r
+END FUNCTION
+
+#    LOADN,      // load number
+FUNCTION loadn(): Number
+    VAR r: Number := 1
+    RETURN r
+END FUNCTION
+
+#    LOADS,      // load string
+FUNCTION loads(): String
+    VAR r: String := "Neon"
+    RETURN r
+END FUNCTION
+
+#    LOADY,      // load bytes
+FUNCTION loady(): Bytes
+    RETURN hexbytes
+END FUNCTION
+
+#    LOADA,      // load array
+FUNCTION loada(): Array<Number>
+     RETURN array
+END FUNCTION
+
+#    LOADD,      // load dictionary
+FUNCTION loadd(): Dictionary<Number>
+    RETURN dictionary
+END FUNCTION
+
+#    LOADP,      // load pointer
+FUNCTION loadp(): POINTER TO class
+    VAR x: POINTER TO class := NIL
+    RETURN x
+END FUNCTION
+
+#    LOADJ,      // load object
+FUNCTION loadj(): Object
+    VAR r: Object := 1
+    RETURN r
+END FUNCTION
+
+#    LOADV,      // load voidptr
+FUNCTION loadv()
+    VAR p1: Handle p1 := NIL
+    VAR p2: Handle p2 := p1
+END FUNCTION
+
+#    STOREB,     // store boolean
+FUNCTION storeb(): Boolean
+    LET r: Boolean := TRUE
+    RETURN r
+END FUNCTION
+
+#    STOREN,     // store number
+FUNCTION storen(): Number
+    LET r: Number := 1
+    RETURN r
+END FUNCTION
+
+#    STORES,     // store string
+FUNCTION stores(): String
+    LET r: String := "Neon"
+    RETURN r
+END FUNCTION
+
+#    STOREY,     // store bytes
+FUNCTION storey(): Bytes
+    LET r: Bytes := hexbytes
+    RETURN r
+END FUNCTION
+
+#    STOREA,     // store array
+FUNCTION storea(): Array<Number>
+    LET r: Array<Number> := array
+    RETURN r
+END FUNCTION
+
+#    STORED,     // store dictionary
+FUNCTION stored(): Number
+    LET di: Dictionary<Number> := dictionary
+    RETURN di["one"]
+END FUNCTION
+
+#    STOREP,     // store pointer
+FUNCTION storep()
+    VAR p: POINTER TO class
+    p := NEW class()
+END FUNCTION
+
+#    STOREJ,     // store object
+FUNCTION storej(): Object
+    LET o: Object := "object"
+    RETURN o
+END FUNCTION
+
+#    STOREV,     // store voidptr
+FUNCTION storev()
+    VAR p: Handle
+    p := NIL
+END FUNCTION
+
+#    NEGN,       // negate number
+FUNCTION negn(): Number
+    RETURN -42
+END FUNCTION
+
+#    ADDN,       // add number
+FUNCTION addn(): Number
+    RETURN 42 + 42
+END FUNCTION
+
+#    SUBN,       // subtract number
+FUNCTION subn(): Number
+    RETURN 42 - 42
+END FUNCTION
+
+#    MULN,       // multiply number
+FUNCTION muln(): Number
+    RETURN 42 * 42
+END FUNCTION
+
+#    DIVN,       // divide number
+FUNCTION divn(): Number
+    RETURN 42 / 42
+END FUNCTION
+
+#    MODN,       // modulo number
+FUNCTION modn(): Number
+    RETURN 42 MOD 21
+END FUNCTION
+
+#    EXPN,       // exponentiate number
+FUNCTION expn(): Number
+    RETURN 42 ^ 3
+END FUNCTION
+
+#    EQB,        // compare equal boolean
+FUNCTION eqb():Boolean
```

Comment:
This and a lot of following functions have strange formatting where there is no space after `:`.

Reply:
Fixed...all of them,